src/main/scala/com/amazon/deequ/analyzers/Analyzer.scala

```diff
@@ -249,6 +256,17 @@ case class NumMatchesAndCount(numMatches: Long, count: Long, override val fullCo
   }
 }
 
+case class AnalyzerOptions(nullBehavior: NullBehavior = NullBehavior.Ignore) {
+  def getNullBehavior(): NullBehavior = {
+    nullBehavior
+  }
+}
+
+object NullBehavior extends Enumeration {
+  type NullBehavior = Value
+  val Ignore, Empty, Fail = Value
```

Comment:
What does `Empty` mean for numbers? If it is only applicable for Strings, should we call it `EmptyString` ?